pkg/apis/pipeline/v1/matrix_types.go

```diff
@@ -129,7 +129,8 @@ func (m *Matrix) validateCombinationsCount(ctx context.Context) (errs *apis.Fiel
 func (m *Matrix) validateParamTypes() (errs *apis.FieldError) {
```

Comment:
maybe we can rename this to `validateParams` because it's now doing more than validating param types only - also validating param name uniqueness

Reply:
sure, renamed it to `validateParams` 👍